### ISSUE TITLE
Accessability Compliance of ightmleditor

### ISSUE THREAD

**Issue by vaishnavimakam** [IgniteUI/ignite-ui#1970]
ightmleditor is not compliant with WAI rules for accessibility.
The tab order and the button names are not readout by the accessability tools. I am using NVDA.

I have tried this on the sample provided on your page - https://www.igniteui.com/html-editor/overview

![image](https://user-images.githubusercontent.com/5100911/61573259-0e811600-aac9-11e9-8f5c-0cd9d3a9a1b7.png) 

If you try tabbing to through the controls, you will see that the tab order is followed only for the buttons and the drop downs are ignored.

The text color and the background color does not have any valid text associated with it. So the assistive tech tools only reads them out as button and does not describe what it does.

The disabled buttons should be skipped from the tabbing order, but with ightmleditor, the disabled buttons also get a focus but you cannot see it on the screen either.

We are using version 18.1.76.
